components/idp.js

```diff
@@ -103,7 +102,26 @@ module.config(function($routeProvider) {
     })
     .when(basePath + '/:identity/keys', {
       title: 'Keys',
-      templateUrl: requirejs.toUrl('bedrock-idp/components/key/keys.html')
+      template:
+        '<div>\
+          <div class="row">\
+            <div class="col-md-12">\
+              <h1 class="headline">Keys</h1>\
+            </div>\
+          </div>\
+          <div class="row">\
+            <div class="col-md-12">\
+              <br-keys br-identity="$resolve.identity"></br-keys>\
+            </div>\
+          </div>\
+        </div>',
```

Comment:
The template above uses bootstrap-specific stuff so we'll need to put it in a separate HTML file so it can be overridden should anyone use another framework.